cdrc/fwv.go

```diff
@@ -126,8 +128,20 @@ func (self *FwvRecordsProcessor) ProcessNextRecord() ([]*engine.CDR, error) {
 	self.processedRecordsNr += 1
 	record := string(buf)
 	for _, cdrcCfg := range self.cdrcCfgs {
-		if passes := self.recordPassesCfgFilter(record, cdrcCfg); !passes {
-			continue
+		if len(cdrcCfg.Filters) == 0 {
```

Comment:
would be more clear if we go with len(filters) != 0 first, since the other option would disappear soon hopefully.